Formula/couchdb.rb

```diff
@@ -30,11 +30,18 @@ class Couchdb < Formula
     depends_on "help2man" => :build
   end
 
+  option "with-geocouch", "Build without GeoCouch spatial index extension"
+
   depends_on "spidermonkey"
   depends_on "icu4c"
   depends_on "erlang"
   depends_on "curl" if MacOS.version <= :leopard
 
+  resource "geocouch" do
+    url "https://codeload.github.com/couchbase/geocouch/tar.gz/couchdb1.3.x"
```

Comment:
For this formula, the archive URL was redirecting to this one. If I didn't replace it for the `resource` usage, I was getting a SHA mismatch error.

I'll try the archive URL again; maybe it was something I was doing wrong locally.